apps/stats/stats-1.py

```diff
@@ -201,7 +201,13 @@ def __printVersion( self, script ) :
 		else :
 			version = "unknown"
 
-		print "Gaffer Version : {version}".format( version = version )
+		versions = (
+			( "From the script", version ),
+			( "Used for stats", Gaffer.About.versionString() ),
```

Comment:
Ivan votes for "Script" and "Current" on the grounds that the file is called a script in the Args and Memory section.